### PR TITLE
doc: use conventional commits

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Branch?       | main for features / current stable version branch for bug fixes 
| Bug fix?      | yes/no 
| New feature?  | yes/no 
| Deprecations? | yes/no
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | api-platform/docs#...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,6 +77,30 @@ When you send a PR, just make sure that:
...
+* `docs`: change in the documentation
+* `spec`: spec change
+* `test`: test-related change
+* `perf`: performance optimization
+* `ci`: CI-related change
+* `chore`: updating dependencies and related changes
+
+Examples:
+
+    fix: Fix something
```
Comment (alanpoulain) [inline CONTRIBUTING.md:94]: Shouldn't we prefer lowercase in the commit title?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -77,6 +77,30 @@ When you send a PR, just make sure that:
...
+
+    feat: Introduce X
+
+    feat!: Introduce Y, BC break
```
Comment (alanpoulain) [inline CONTRIBUTING.md:98]: It would be nice to add an example with a scope, like:
```
feat(metadata): add a metadata feature
```

**Comment by guilliamxavier** [comment]
Si your commit message should starts with `docs:` not `doc:` 😄 (also the PR description is still template)

File: CONTRIBUTING.md (hunk 2 above)
Comment (dunglas) [reply CONTRIBUTING.md:98]: We should even make the scope mandatory for Core.

**Pull request merged**